ios/AppLovinMAXAdViewUIComponent.m

```diff
@@ -122,41 +122,55 @@ - (void)destroy
 
 - (void)didLoadAd:(MAAd *)ad
 {
-    NSDictionary *adInfo = [[AppLovinMAX shared] adInfoForAd: ad];
+    NSMutableDictionary *body = [@{@"adViewId": @(self.adView.hash)} mutableCopy];
+    [body addEntriesFromDictionary: [[AppLovinMAX shared] adInfoForAd: ad]];
     
-    [[AppLovinMAX shared] sendEventWithName: @"OnNativeUIComponentAdViewAdLoadedEvent" body: adInfo];
+    if ( [AppLovinMAXAdView preloadedAdView: @(self.adView.hash)] )
+    {
+        [[AppLovinMAX shared] sendEventWithName: @"OnNativeUIComponentAdViewAdLoadedEvent" body: body];
```

Comment:
Should we use constant for this? `ON_NATIVE_UI_COMPONENT_ADVIEW_AD_LOADED_EVENT`